java/client/src/main/java/glide/connectors/resources/Platform.java

```diff
@@ -38,32 +45,56 @@ public static class Capabilities {
     /** Detect <em>kqueue</em> availability. */
     private static boolean isKQueueAvailable() {
         try {
+            debugInfo += "Checking KQUEUE...\n";
```

Comment:
Can we make this functionality ONLY when debug mode is enabled (debug log level is enabled)?

Reply:
Added